### PR TITLE
Adjust logging level in kube-up to avoid very detailed http logs

### PR DESCRIPTION
**Pull request by dims**
Reviewing the log files (for example those in https://github.com/kubernetes/kubernetes/issues/109038#issuecomment-1079602652) more explicitly this one:
https://gcsweb.k8s.io/gcs/kubernetes-jenkins/pr-logs/pull/109061/pull-kubernetes-integration/1508163205329850368/artifacts/

The size of the log file is too much! wading through the stdout log for example to see which file/line logged the most number of times, we see `httplog.go:131` as a big culprit. 

sample line:
```
{"Time":"2022-03-27T19:57:45.884101219Z","Action":"output","Package":"k8s.io/kubernetes/test/integration/service","Test":"Test_ExternalNameServiceDropsInternalTrafficPolicy","Output":"I0327 19:57:45.884025  127031 httplog.go:131] \"HTTP\" verb=\"POST\" URI=\"/apis/flowcontrol.apiserver.k8s.io/v1beta2/prioritylevelconfigurations?fieldManager=api-priority-and-fairness-config-producer-v1\" latency=\"4.893606ms\" userAgent=\"service.test/v0.0.0 (linux/amd64) kubernetes/$Format\" audit-ID=\"18871a17-78dc-4458-9035-82dcb8b116dd\" srcIP=\"127.0.0.1:58142\" apf_pl=\"exempt\" apf_fs=\"exempt\" resp=201\n"}
```

here's how i grep-ped the lines:
```
rg '([0-9a-z_]+\.go:[0-9]+)' -or '$1' ./1508163205329850368/artifacts/junit_20220327-194613.stdout | cut -f 1- -d ':' | sort | uniq -c | sort -n
```

the tail end of the output:
```
6682 instance.go:599
7243 mutation_detector.go:53
7451 pv_controller.go:542
7528 pv_controller.go:872
7736 genericapiserver.go:538
7848 apf_controller.go:444
8148 strategy.go:235
9683 pv_controller.go:869
9771 apf_controller.go:633
10888 factory.go:134
16039 storage_rbac.go:257
16527 pv_controller_base.go:689
21768 store.go:1457
22700 cacher.go:410
22700 watch_cache.go:566
23126 storage_rbac.go:227
23136 warnings.go:70
26340 reflector.go:255
32021 healthz.go:257
40565 shared_informer.go:285
43295 retry_interceptor.go:62
66477 storage_factory.go:286
214398 httplog.go:131
```

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by dims** [comment]
cc @pohly

**Comment by aojea** [comment]
default level is v2

> https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use

for CI I found these logs useful

File: staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go [hunk 1]
```diff
@@ -95,7 +95,9 @@ func DefaultStacktracePred(status int) bool {
 	return (status < http.StatusOK || status >= http.StatusInternalServerError) && status != http.StatusSwitchingProtocols
 }
 
-const withLoggingLevel = 3
+// default logging level INFO is 4. Let's make sure if folks really need the http logs they
```
Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go:98]: I don't think default log level is 4, I think it's 2. It gets bumped up in test, right?

File: cluster/gce/config-test.sh [hunk 2]
```diff
@@ -207,7 +207,7 @@ export ETCD_DOCKER_REPOSITORY=${TEST_ETCD_DOCKER_REPOSITORY:-}
 export ETCD_VERSION=${TEST_ETCD_VERSION:-}
 
 # Default Log level for all components in test clusters and variables to override it in specific components.
-TEST_CLUSTER_LOG_LEVEL=${TEST_CLUSTER_LOG_LEVEL:---v=4}
+TEST_CLUSTER_LOG_LEVEL=${TEST_CLUSTER_LOG_LEVEL:---v=2}
```
Comment (liggitt) [inline cluster/gce/config-test.sh:210]: this doesn't control integration tests, right?

Comment (liggitt) [reply cluster/gce/config-test.sh:210]:
```
// MinVerbosity determines the minimum klog verbosity when running tests that
// involve the apiserver.  This overrides the -v value from the command line,
// i.e. -v=0 has no effect when MinVerbosity is 4 (the default).  Tests can opt
// out of this by setting MinVerbosity to zero before starting the control
// plane or choose some different minimum verbosity.
var MinVerbosity = 4
```

I think that's where this is getting set in integration

Comment (dims) [reply cluster/gce/config-test.sh:210]: @liggitt going to check that in the current CI job that is running. will let you know.

Comment (liggitt) [reply cluster/gce/config-test.sh:210]: I'm confident cluster scripts aren't used in integration

Comment (dims) [reply cluster/gce/config-test.sh:210]: updated PR, there's a few more places.

Comment (liggitt) [reply cluster/gce/config-test.sh:210]: unless verbosity level is causing us issues in e2e tests, I'd really like to not drop verbosity there... logs are the main way we can piece together issues that reproduce rarely or only in the CI env

Comment (dims) [reply cluster/gce/config-test.sh:210]: logs show here's how it is set.
```
+ popd
+ export KUBE_COVER=n
+ KUBE_COVER=n
+ export ARTIFACTS=/logs/artifacts
+ ARTIFACTS=/logs/artifacts
+ export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
+ KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
+ export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
+ KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
+ export LOG_LEVEL=4
+ LOG_LEVEL=4
+ cd /home/prow/go/src/k8s.io/kubernetes
```

File: staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go (hunk 1 above)
Comment (dims) [reply staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go]: you are right. fixed in latest PR

**Comment by dims** [comment]
closing, let's revisit if this becomes a bigger problem.

**Comment by liggitt** [comment]
> default level is v2
> 
> > https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#what-method-to-use
> 
> for CI I found these logs useful

For e2e I definitely agree. For integration/units, I could see lowering to 2 or 3 (probably 3) by default, since those are easy to run locally with `stress` to reproduce flakes

**Comment by pohly** [comment]
Is changing the verbosity level of integration tests this still relevant?

The default verbosity was increased to 4 [three years ago](https://github.com/kubernetes/kubernetes/commit/e6f8e51c501dbdd82203a03de58dcfca6eacdaca). I made that [configurable for performance tests](https://github.com/kubernetes/kubernetes/commit/e1e84c8e5f56d0f6e709e418839662200243ec90) but left the default that is used by most tests as it was.

Lowering that default would be easy: https://github.com/kubernetes/kubernetes/blob/e0ca5cfd73bd046ee5fbdef50e073842e6fbc52f/test/integration/framework/controlplane_utils.go#L73

**Pull request closed without merging**